tests/unit/test_names.c

```diff
@@ -274,15 +271,24 @@ int check_strerror_pio(int my_rank)
 int check_strerror(int my_rank)
 {
     int ret;
+
+    printf("checking strerror for netCDF-classic error codes...\n");
```

Comment:
Should all of these printf's be here?  That's going to be pretty noisy isn't it?